data/tlg1286/tlg001/tlg1286.tlg001.1st1K-grc1.xml

```diff
@@ -0,0 +1,543 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+   <teiHeader xml:lang="eng">
+       <fileDesc>
+           <titleStmt>
+               <title xml:lang="grc">[Ἑρμοῦ τρισμεγίστου] Ποιμάνδρης</title>
+               <author>Hermes Trimegistus</author>
```

Comment:
Probably should be Corpus Hermeticum, but if kept as is, there is an "s" missing in the name.